File: CISC 131 c/score_to_grade.c
#include<stdio.h>

int num = 87;
char toLetterGrade(int num);
char toLetterGrade2(int num);

int main(){
    printf("Grade: %c", toLetterGrade(num));
    return 0;
}

char toLetterGrade(int num){
    if (num >= 90){
        return 'A';
    }
    else if (num < 60){
        return 'F';
    }
    else if (num >= 80){
        return 'B';
    }
    else if (num >= 70){
        return 'C';
    }
    else{
        return 'D';
    }
}

char toLetterGrade2(int num){
    if (num < 60){
        return 'F';
    }
    else{
        int ten_digit = num/10;
        int ascii_val = 74 - ten_digit;
        return ascii_val;
    }
}

File: CISC 131 py/score_to_grade.py
def toLetterGrade(num):
    if num >= 90:
        return "A"
    elif num < 60:
        return "F"
    elif num >= 80:
        return "B"
    elif num >= 70:
        return "C"
    else:
        return "D"

def toLetterGrade2(num):
    if num < 60:
        return 'F'
    else:
        ten_digit = int(num/10)
        ascii_val = 74 - ten_digit
        return chr(ascii_val)

def main():
    print(toLetterGrade(92))
    print(toLetterGrade(83))
    print(toLetterGrade(59))
    print(toLetterGrade(60))
    print(toLetterGrade(72))

    print()

    print(toLetterGrade2(92))
    print(toLetterGrade2(83))
    print(toLetterGrade2(59))
    print(toLetterGrade2(60))
    print(toLetterGrade2(72))

main()
